Demo/UITests/BraintreeDropIn_Localization_UITests.swift

```diff
@@ -18,7 +18,7 @@ class BraintreeDropIn_Localization_UITests: XCTestCase {
         app.buttons["Add Payment Method"].tap()
 
         expectation(for: NSPredicate(format: "exists == true"), evaluatedWith: app.staticTexts["Seleccionar forma de pago"])
-        waitForExpectations(timeout: 10.0)
+        waitForExpectations(timeout: 20)
```

Comment:
Dang, were these really timing out? 10s is a lot on it's own

Reply:
Yeah some of them were legit timing out looking through the logs 😢

Reply:
That is a lot for a UI test unless it's making networking calls or performing disk i/o.